src/Service/OS2Forms/CaseSubmissionManager/OS2FormsManager.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Service\OS2Forms\CaseSubmissionManager;
+
+class OS2FormsManager
+{
+    public function __construct(private ResidentComplaintBoardCaseTypeManager $residentComplaintBoardCaseTypeManager)
+    {
+    }
+
+    public function getOS2FormsCaseManagerFromWebformId(string $webformId)
+    {
+        // TODO: Needs updating when form ids are finalized.
+        $manager = match ($webformId) {
```

Comment:
You don't need this variable – just return the value.

Reply:
Fixed.